docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'How to increase the findability, visibility, and impact of Galaxy tools for your scientific community'
+title_short: 'BioHackEU23 project #25: Finding Galaxy tools'
+tags:
+  - Findability
+  - Galaxy
+  - Community specific
+  - Tools
+  - EDAM
+  - bio.tools
+  - Metadata
+authors:
```

Comment:
We should probably add everyone who helped during the BioHackathon as authors. What do you think?

Reply:
Yes, we should ask everyone that was part of the project if they would like to be added as authors.

Reply:
I just asked the BH channel, please also contact everyone you think should be added.

ping:

@bgruening 
@mvdbeek

Reply:
Thanks for asking but I think my contributions are to minor to be listed here.

Reply:
Same, thanks for thinking of me, but I don't think I did enough to be on the paper.

Reply:
Can I add myself to this preprint even if I didn't contribute that much?
I made a PR here https://github.com/supernord/galaxy_tool_extractor/pull/2
@supernord @paulzierep

Reply:
@J-Swang yes please add yourself, you helped !

Reply:
- [ ] @EngyNasr 
- [ ] Australian BioCommons folks

Reply:
Following up with the additional Australian outpost members now

Reply:
I only made my contribution during the Biohackaton 2023 not to the paper, so I am not sure if that counts as contribution.

Reply:
> I only made my contribution during the Biohackaton 2023 not to the paper, so I am not sure if that counts as contribution.

Of course it does!